### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wulfland**

### CODE REVIEW

**Comment by wulfland** [comment]
Hey @kaufm

File: docs/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Javascript Tetris
 =================
 
+Hello World!≈
+
```
Comment (kaufm) [inline docs/README.md:5]:
```suggestion
Hello World! ;)

```

File: docs/README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Javascript Tetris
 =================
 
+Hello World! ;)
```
Comment (kaufm) [inline docs/README.md:4]: 🏅 Looks great

**Review by kaufm — APPROVED** [review]
👍 Looks great

**Pull request merged**